docs/concepts/content-types.md

```diff
@@ -134,7 +134,7 @@ Use to send a read receipt, which is a `timestamp` that indicates when a message
 Use a reaction to send a quick and often emoji-based way to respond to a message. Reactions are usually limited to a predefined set of emojis or symbols provided by the messaging app.
 
 - [Read the doc](/docs/build/messages/reaction)
-- [Comment on the XIP idea](https://github.com/orgs/xmtp/discussions/36)
+- [Comment on the XIP idea](https://community.xmtp.org/t/proposal-for-emoji-reactions-content-type/499?u=jha)
```

Comment:
is the `u=jha` required in this link?

Reply:
Thank you for this feedback, eagle eye @rygine! Fixing now...... <3